modules/installation-adding-registry-pull-secret.adoc

```diff
@@ -0,0 +1,112 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_restricted_networks/installing-restricted-networks-preparations.adoc
+
+[id="installation-adding-registry-pull-secret_{context}"]
+= Adding the registry to your pull secret
+
+Modify your the pull secret for your {product-title} cluster to describe
+your local registry before you install an {product-title} cluster in a
+restricted network.
+
+.Prerequisites
+
+* You configured a mirror registry to use in your restricted network.
+
+.Procedure
+
+Complete the following steps on the bastion host:
+
+. Download your `registry.redhat.io` pull secret from the
+link:https://cloud.redhat.com/openshift/install[OpenShift Infrastructure Providers]
+page.
+
+. Generate the base64-encoded user name and password or token for your mirror
+registry:
++
+----
+$ echo -n '<user_name>:<password>' | base64 -w0 <1>
+
+BGVtbYk3ZHAtqXs=
+----
+<1> For `<user_name>` and `<password>`, specify the user name and password that
+you configured for your registry.
+
+. Make a copy of your pull secret in JSON format:
++
+----
+$ cat ./pull-secret.text | jq .  > <path>/<pull-secret-file><1>
+----
+<1> Specify the path to the folder to store the pull secret in and a name for
+the JSON file that you create.
++
+The contents of the file resemble the following example:
++
+----
+{
+  "auths": {
+    "cloud.openshift.com": {
+      "auth": "b3BlbnNo...",
+      "email": "you@example.com"
+    },
+    "quay.io": {
+      "auth": "b3BlbnNo...",
+      "email": "you@example.com"
+    },
+    "registry.connect.redhat.com": {
+      "auth": "NTE3Njg5Nj...",
+      "email": "you@example.com"
+    },
+    "registry.redhat.io": {
+      "auth": "NTE3Njg5Nj...",
+      "email": "you@example.com"
+    }
+  }
+}
+----
+
+. Edit the new file and add a section that describes your registry to it:
++
+----
+  "auths": {
+...
+    "<bastion_host_name>:<local_registry_host_port>": { <1>
+      "auth": "<credentials>", <2>
+      "email": "you@example.com"
+  },
+...
+----
+<1> For `bastion_host_name`, specify the registry domain name
+that you specified in your certificate, and for `<local_registry_host_port`,
```

Comment:
nit: `<local_registry_host_port` -> `<local_registry_host_port>`.  And you can leave off the `:port` if you're using the standard HTTP(S) of 80/443, although it may not be worth calling that out.